pum/core/upgrader.py

```diff
@@ -193,7 +202,11 @@ def __run_sql_file(self, filepath):
             the path of the file to execute"""
 
         with open(filepath, 'r') as delta_file:
-            self.cursor.execute(delta_file.read())
+            sql = delta_file.read()
+            for var in self.variables or ():
+                p = re.compile(r':{}\b'.format(var[0]))
```

Comment:
Out of curiosity is there any specific reason for choosing the `:var` notation?

Reply:
that is the notation used by `psql` to pass variables (you have quite some in qwat for SRIDs).
It would have been logic, but it is not as safe and does not offer type control as `psycopg` does

Reply:
@3nids I since realized that you changed this in #42.